### DISCUSSION TITLE
Zustand 가 리덕스에 영감을 만들어 졌다는데 둘 다 사용해신 분이있다면 공통점과 차이점이 궁금합니다.

### DISCUSSION THREAD

**Discussion by wogus216** [diving-react/study-react-deepdive#83] [General]
Zustand 가 리덕스에 영감을 만들어 졌다는데 둘 다 사용해신 분이있다면 공통점과 차이점이 궁금합니다.

**Comment by HYEOK999** [comment]
redux는 보일러플레이트가 굉장히 심합니다.
redux측에서도 그 부분을 인지하고, redux-toolkit이라고 내놓았습니다.

1세대 상태관련라이브러리인 redux는 사용해보시면 굉장히 무겁고 러닝커브도 있습니다.
3세대 zustand의 경우, redux를 사용한 가장 큰 이유 중 하나인 redux-dev-tools 를 사용할 수 있을 뿐더러, 굉장히 가볍고 컨셉자체도 굉장히 간단하게 되어있습니다.
